### DISCUSSION TITLE
report error of storage_p.buffer.targets function

### DISCUSSION THREAD

**Discussion by Amos58** [ContinualAI/avalanche#1604] [Q&A]
I am reproduce code from https://avalanche.continualai.org/how-tos/dataloading_buffers_replay. When I run "storage_p.buffer.targets",  it shows that 
![image](https://github.com/ContinualAI/avalanche/assets/55913276/b8c283fe-360d-4455-95c9-319ebcdbda6b)

And here is the whole code :
![image](https://github.com/ContinualAI/avalanche/assets/55913276/daf22979-4c6c-4400-a390-5df86b323023)

my avalanche version is 0.3.1

**Comment by AntonioCarta** [comment]
This looks fixed in the latest version (0.5.0).
